pkg/kubelet/cm/memorymanager/policy_static.go

```diff
@@ -382,7 +469,7 @@ func getRequestedResources(container *v1.Container) (map[v1.ResourceName]uint64,
 	return requestedResources, nil
 }
 
-func (p *staticPolicy) calculateHints(s state.State, requestedResources map[v1.ResourceName]uint64) map[string][]topologymanager.TopologyHint {
+func (p *staticPolicy) calculateHints(s state.State, requestedResources map[v1.ResourceName]uint64, reusable map[v1.ResourceName]state.Block) map[string][]topologymanager.TopologyHint {
```

Comment:
we should check that `reusable != nil` before use it

Reply:
true, there are checks now in proper places.